crates/rolldown/src/module_finalizers/scope_hoisting/impl_visit_mut.rs

```diff
@@ -302,6 +329,9 @@ impl<'ast> VisitMut<'ast> for ScopeHoistingFinalizer<'_, 'ast> {
         }
       }
     } else {
+      if let Some(ref_id) = self.try_get_valid_namespace_alias_ref_id_from_member_expr(expr) {
```

Comment:
If this method could recognize the pattern here, why we don't just do the rewriting here?

It doesn't need to maintain `interested_namespace_alias_ref_id` and change the definition of finalized_expr_for_symbol_ref anymore.

Reply:
Then we have two position to rewrite `IdentifierReference`， It may reduce the maintainability